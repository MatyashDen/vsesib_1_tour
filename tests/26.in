28 7 7 7
0 3
0 4
3 6
10 6
11 4
17 0
20 0
