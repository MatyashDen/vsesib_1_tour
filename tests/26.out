908
0LRE0DUE0LLRRE0LDURE0DDUUE0LLLRRRE0LLDURRE0LDDUURE0DDDUUUE0LLLDURRRE0LLDDUURRE0LDDDUUURE0DDDDUUUUE0LLLDDUURRRE0LLDDDUUURRE0LDDDDUUUURE0DDDDDUUUUUE0LLLDDDUUURRRE0LLDDDDUUUURRE0LDDDDDUUUUURE0DDDDDDUUUUUUE0LLLDDDDUUUURRRE0LLDDDDDUUUUURRE0LDDDDDDUUUUUURE0LLLDDDDDUUUUURRRE0LLDDDDDDUUUUUURRE0LLLDDDDDDUUUUUURRREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
1RLE1DUE1RRLLE1RDULE1DDUUEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
2LRE2DUE2UDE2LLRRE2LDURE2LUDRE2DDUUE2UUDDE2LLDURRE2LDDUURE2DDDUUUE2LLDDUURRE2LDDDUUURE2LLDDDUUURREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
3LRE3DUE3UDE3LUDRE3DDUUE3UUDDE3LUUDDRE3DDDUUUE3UUUDDDE3LUUUDDDRE3DDDDUUUUE3DDDDDUUUUUE3DDDDDDUUUUUUEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
4RLE4LRE4DUE4UDE4RDULE4LLRRE4LDURE4LUDRE4DDUUE4UUDDE4RDDUULE4LLLRRRE4LLDURRE4LLUDRRE4LDDUURE4LUUDDRE4DDDUUUE4UUUDDDE4RDDDUUULE4LLLLRRRRE4LLLDURRRE4LLLUDRRRE4LLDDUURRE4LLUUDDRRE4LDDDUUURE4LUUUDDDRE4DDDDUUUUE4UUUUDDDDE4RDDDDUUUULE4LLLLDURRRRE4LLLLUDRRRRE4LLLDDUURRRE4LLLUUDDRRRE4LLDDDUUURRE4LLUUUDDDRRE4LDDDDUUUURE4LUUUUDDDDRE4DDDDDUUUUUE4RDDDDDUUUUULE4LLLLUUDDRRRRE4LLLUUUDDDRRRE4LLUUUUDDDDRRE4LLLLUUUDDDRRRRE4LLLUUUUDDDDRRRE4LLLLUUUUDDDDRRRREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
5RLE5DUE5UDE5RRLLE5RDULE5RUDLE5UUDDE5RRRLLLE5RRDULLE5RRUDLLE5RUUDDLE5UUUDDDE5RRRRLLLLE5RRRDULLLE5RRRUDLLLE5RRUUDDLLE5RUUUDDDLE5UUUUDDDDE5RRRRRLLLLLE5RRRRDULLLLE5RRRRRRLLLLLLE5RRRRRDULLLLLE5RRRRRRDULLLLLLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
6RLE6DUE6UDE6RRLLE6RDULE6RUDLE6DDUUE6RRRLLLE6RRDULLE6RRUDLLE6RDDUULE6DDDUUUE6RRRRLLLLE6RRRDULLLE6RRRUDLLLE6RRDDUULLE6RDDDUUULE6DDDDUUUUE6RRRRRLLLLLE6RRRRDULLLLE6RRRRUDLLLLE6RRRDDUULLLE6RRDDDUUULLE6RDDDDUUUULE6DDDDDUUUUUE6RRRRRRLLLLLLE6RRRRRDULLLLLE6RRRRRUDLLLLLE6RRRRDDUULLLLE6RRRDDDUUULLLE6RRDDDDUUUULLE6RDDDDDUUUUULE6DDDDDDUUUUUUE6RRRRRRDULLLLLLE6RRRRRRUDLLLLLLE6RRRRRDDUULLLLLE6RRRRDDDUUULLLLE6RRRDDDDUUUULLLE6RRDDDDDUUUUULLE6RDDDDDDUUUUUULE6DDDDDDDUUUUUUUE6RRRRRRDDUULLLLLLE6RRRRRDDDUUULLLLLE6RRRRDDDDUUUULLLLE6RRRDDDDDUUUUULLLE6RRDDDDDDUUUUUULLE6RDDDDDDDUUUUUUULE6RRRRRRDDDUUULLLLLLE6RRRRRDDDDUUUULLLLLE6RRRRDDDDDUUUUULLLLE6RRRDDDDDDUUUUUULLLE6RRDDDDDDDUUUUUUULLE6RRRRRRDDDDUUUULLLLLLE6RRRRRDDDDDUUUUULLLLLE6RRRRDDDDDDUUUUUULLLLE6RRRDDDDDDDUUUUUUULLLE6RRRRRRDDDDDUUUUULLLLLLE6RRRRRDDDDDDUUUUUULLLLLE6RRRRDDDDDDDUUUUUUULLLLE6RRRRRRDDDDDDUUUUUULLLLLLE6RRRRRDDDDDDDUUUUUUULLLLLE6RRRRRRDDDDDDDUUUUUUULLLLLLE
